src/components/CommentInfo.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Comment } from './types/Comment';
+
+type Props = {
+  comments: Comment,
+};
+
+export const CommentInfo: React.FC<Props> = ({ comments }) => (
```

Comment:
This is one comment object, not array
```suggestion
type Props = {
  comment: Comment,
};

export const CommentInfo: React.FC<Props> = ({ comment }) => (
```